<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Алфавит (сплошняком)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Подцепочка без пробелов</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Кратность (без пробелов)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>300</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Таблица переходов</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>160</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Сгенерировать ДКА</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>160</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Сген. изобр. ДКА</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>160</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Проверить цепочку</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Цепочка</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>340</y>
      <width>400</width>
      <height>68</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>580</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ошибок нет</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>400</width>
      <height>248</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>580</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Как же я задолбался писать этот интерфейс, вы бы знали, я не спал до 3 часов сегодня кабзда</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>620</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <action name="actionAuthor">
   <property name="text">
    <string>Author</string>
   </property>
  </action>
  <action name="actionTask">
   <property name="text">
    <string>Task</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
